saleor/discount/models.py

```diff
@@ -454,6 +454,9 @@ class BaseDiscount(models.Model):
         choices=DiscountType.CHOICES,
         default=DiscountType.MANUAL,
     )
+    # `unique_type` is a field to ensure, that only single specific discount type
+    # can be associated with discounted object
```

Comment:
Maybe worth to mention that we are adding it as a separate field, as previously there was such possibility, and starting from Saleor 3.X we want to fully block it on database.

Reply:
Description updated